hie-plugin-api/Haskell/Ide/Engine/Ghc.hs

```diff
@@ -272,8 +272,8 @@ setTypecheckedModule_load uri =
 
             Session sess <- GhcT pure
             modifyMTS (\s -> s {ghcSession = Just sess})
-            cacheModules rfm ts
-            --cacheModules rfm [tm]
+--            cacheModules rfm ts
```

Comment:
`ts` is unused now, right? This will cause the CI to fail, if it isnt prefixed with `_`